src/process_utility.c

```diff
@@ -333,6 +333,42 @@ process_drop_foreign_server_start(DropStmt *stmt)
 	}
 }
 
+static void
+process_drop_trigger_start(ProcessUtilityArgs *args, DropStmt *stmt)
```

Comment:
So, creation happens manually or via distributed exec. But drop is handled internally?